crypto/src/vrf/schnorrkel/mod.rs

```diff
@@ -103,6 +103,23 @@ impl SchnorrkelPrivateKey {
         (sk, pk)
     }
 
+    /// Derive a key from a 32-byte secret
+    pub fn new_from_bytes(
+        bytes: &[u8],
+    ) -> Result<(SchnorrkelPrivateKey, SchnorrkelPublicKey), VRFError> {
+        let mini_secret = schnorrkel::MiniSecretKey::from_bytes(bytes)
+            .map_err(|e| VRFError::GenerateKeyError(e.to_string()))?;
+        let keypair = mini_secret.expand_to_keypair(schnorrkel::ExpansionMode::Uniform);
+        Ok((
+            SchnorrkelPrivateKey {
+                key: keypair.secret.clone(),
+            },
+            SchnorrkelPublicKey {
+                key: keypair.public,
+            },
+        ))
+    }
```

Comment:
This has to go.

Reply:
I think we can keep this as it was discussed?